examples/angular/src/lib/todos/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")
+load("//tools:angular_ts_library.bzl", "ng_ts_library")
+
+package(default_visibility = ["//:__subpackages__"])
+
+sass_binary(
+    name = "todos-styles",
+    src = "todos.component.scss",
+)
+
+ng_ts_library(
+    name = "todos",
+    srcs = [
+        "todos.component.ts",
+        "todos.module.ts",
+    ],
+    angular_assets = [
+        "todos.component.html",
+        ":todos-styles",
+    ],
+    module_name = "@lib/todos",
```

Comment:
Adding this fails, from what I can read this might not be the right way to do it, but what is the right way then?